src/main/java/teammates/storage/search/FeedbackResponseCommentSearchDocument.java

```diff
@@ -321,6 +321,12 @@ public static FeedbackResponseCommentSearchResultBundle fromResults(
             bundle.commentGiverTable.put(comment.getId().toString(),
                     getFilteredCommentGiverName(bundle, instructorCourseIdList, response, comment, commentGiverName));
             bundle.instructorEmailNameTable.put(comment.giverEmail, commentGiverName);
+            if (!comment.lastEditorEmail.isEmpty()) {
+                InstructorAttributes instructor = instructorsDb.getInstructorForEmail(response.courseId,
```

Comment:
I am thinking of changing it like this:-
Check if map already has email, if not make a Db call.
Will this be fine?